nrf5340-net-hal/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "nrf5340-net-hal"
+version = "0.1.0"
+description = "HAL for nRF5340 net SoC"
+readme = "../README.md"
+
+repository = "https://github.com/nrf-rs/nrf-hal"
+authors = [
+    "Jonathan Pallant (42 Technology) <jonathan.pallant@42technology.com>",
+    "Sascha Wise <me@saschawise.com",
```

Comment:
Authors list doesn't look correct. You can either list yourself there, or remove the `authors` field, since it's optional.